.github/workflows/conda-build.yml

```diff
@@ -18,10 +18,10 @@ jobs:
           auto-activate-base: true
           activate-environment: true
           python-version: ${{ matrix.python-version }}   
-    - name: Build Bystro using Maturin
```

Comment:
Testing whether we can install is more useful than whether we can build the wheel (both are tested here, since the wheel is made during the build-python step, which is automatically run as part of the install step)